test/client-unit/services/PasswordMeterService.spec.js

```diff
@@ -75,12 +74,6 @@ describe('PasswordMeterService', () => {
     expect(validate).to.equal(true);
   });
 
-  it('#validate() should return true if the session is not set', () => {
```

Comment:
Instead of deleting tests to make the test pass, can you alter this test to show the new behavior?